source/about/mattermost-server-releases.md

```diff
@@ -19,6 +19,7 @@ Mattermost releases a new server version on the 16th of each month in [binary fo
 
 | **Release** | **Released on** | **Support ends** |
 |:---|:---|:---|
+| v10.0 [Download](https://releases.mattermost.com/10.0.0/mattermost-10.0.0-linux-amd64.tar.gz) \| {ref}`Changelog <release-v10-0-major-release>` | 2024-09-16 | 2024-12-15 |
```

Comment:
@amyblais  I think this is a typo.  Support ends in December?

Reply:
All non-ESR releases are supported for 3 months.